packages/grid/src/GridMetricCalculator.ts

```diff
@@ -1250,19 +1706,19 @@ class GridMetricCalculator {
 
   /**
    * Resets the column width for the specified column to the calculated width
-   * @param {Number} column The column model index to reset
+   * @param column The column model index to reset
    */
-  resetColumnWidth(column) {
-    this.setColumnWidth(column, null);
+  resetColumnWidth(column: ModelIndex): void {
+    this.setColumnWidth(column, undefined);
     this.calculatedColumnWidths.delete(column);
   }
 
   /**
    * Sets the width for the specified row
-   * @param {Number} row The row model index to set
-   * @param {Number} size The size to set it to, or null to reset the row size
+   * @param row The row model index to set
+   * @param size The size to set it to, or null to reset the row size
    */
-  setRowHeight(row, size) {
+  setRowHeight(row: ModelIndex, size?: number): void {
```

Comment:
Same issue as above w/ setting to null not being allowed by types and actually deleting the value